packages/server/lib/browsers/chrome.ts

```diff
@@ -55,6 +55,18 @@ let browserCriClient: BrowserCriClient | undefined
  * @param userDir
  */
 const _getChromePreferences = (userDir: string): Bluebird<ChromePreferences> => {
+  /**
+   * By returning empty objects here, the user is indicating that they do not want
+   * Chrome preferences read. If they do not provide preferences via config,
+   * this effectively prevents Cypress from writing to the Chrome preferences files.
+   * See: _writeChromePreferences
+   */
+  if (process.env.IGNORE_CHROME_PREFERENCES) {
+    debug('ignoring chrome preferences...')
+
+    return Bluebird.resolve(_.mapValues(CHROME_PREFERENCE_PATHS, () => ({})))
```

Comment:
What if only one of the `CHROME_PREFERENCE_PATHS` is encrypted, would we want to read from the other paths?

Reply:
I don't think so - this makes more sense as an all-or-nothing thing. I'm curious if we should prevent writing preferences as well with this env var, but that may cause more confusion (or should be a separate env var). As is, if you set this env, and then provide preferences via config, it will still attempt to write preferences.